package.json

```diff
@@ -50,6 +52,7 @@
     "commitizen": "4.0.3",
     "core-js": "3.6.4",
     "fast-deep-equal": "3.1.1",
+    "ngx-observable-lifecycle": "1.0.1",
```

Comment:
Don't forget to add these to the dependencies in `projects/ngx-sub-form`, right now I get errors if I don't install this and `tsdef` myself:
```
ERROR in The target entry-point "ngx-sub-form/new" has missing dependencies:
 - ngx-observable-lifecycle
 - tsdef
```

Reply:
Good point, will do thanks!